guides/tutorials/codelabs/azure-vmss-source-to-prod/index.md

```diff
@@ -0,0 +1,326 @@
+---
+layout: single
+title:  "Azure VM Scale Sets: Source to Prod"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+In this codelab, you will create a source-to-prod continuous delivery flow from source to production for a simple java based application.
+
+After completing this lab the following workflow will happen: 
+  1. Some code is checked in on gthub.
+  2. Jenkins automatically runs a job to create a new build. 
+  3. Spinnaker deploys the new version of the application to a Test server group and disable the previous deployment.
+  4. The deployment is verified
+  5. Spinnaker redeploys the same image to the production environment and disable the previous deployment.
+  
+This codelab will walk through the steps to  implement this scenario: 
+
+  * Setup the environment and the necessary pre-requisites
+  * Configure Jenkins and Spinnaker
+  * Create the pipeline for the deployment
+
+## 0. Environment pre-requisites 
+
+### Deploy Spinnaker and Jenkins on Azure 
+
+You need the following resources to perform the codelab: 
+- **Azure subscription**: You can create one for free using the [Azure Free trial](https://azure.microsoft.com/free/).
+- **Spinnaker and Jenkins VM**: You can deploy them on your Azure subscription with the [Continuous Deployment quick start template](https://github.com/Azure/azure-quickstart-templates/tree/master/301-jenkins-aptly-spinnaker-vmss) 
+
+From a shell run the following command that will walk you through all the steps necessary to deploy Jenkins and Spinnaker in Azure:
+
+```bash 
+bash <(curl -sL https://aka.ms/DeployDevOps)
+```
+
+In the output of the above script you will find the command to open an SSH tunnel to the DevOps VM that has just been deployed. The command should look like this: 
+
+```bash
+ssh -L 8080:localhost:8080 -L 9000:localhost:9000 -L 8084:localhost:8084 -L 8087:localhost:8087 username@vmdnsprefix.selected_region.cloudapp.azure.com
+```
+
+> NOTE: You will have to enter the password that you have specified in the previous step.
+
+
+## 1. Configure Jenkins 
+
+1. After you have started your ssh tunnel, navigate to `http://localhost:8080/` on your local machine.
+1. The instance should already be unlocked and your first account setup. Login with the credentials you specified when deploying the template.
+1. Click **Install suggested plugins**: 

```

Comment:
You have to select one of the two options on this page. The "install suggest plugins" is the option that is the most straightforward.